src/background/telemetry.js

```diff
@@ -14,6 +14,9 @@ const TELEMETRY_CATEGORY = "lockboxv1";
 //  which seems like a code smell, while the action / middleware approach solves
 //  this nicely.
 
+let enabled = false;
+function getTelemetryEnabled() { return enabled; }
+
 function registerEvents() {
```

Comment:
nit: since there's an enabled check at the start of all the other telemetry functions, you could add one at the top of `registerEvents` as well, just for uniformity

Reply:
I put the enabled check on all the exported functions, figuring that would be sufficient.